### DISCUSSION TITLE
网站访问正常但是列表提示ERROR

### DISCUSSION THREAD

**Discussion by OrdChaos** [travellings-link/travellings#2471] [寻求帮助] [answered]
如题
url: https://www.ordchaos.com
已经连续超过两周，一直提示Axios error: getaddrinfo EAI_AGAIN www.ordchaos.com

**Comment by gxres042** [comment]
哥们要不自查一下你开了什么逆天的设定，导致巡查机器人用 Axios 检查的时候被拦了
我挂个魔法访问你网站都解析不出来

![Screenshot_2024-08-15-18-24-46-760_com.android.chrome.jpg](https://github.com/user-attachments/assets/852f117c-cc13-4649-a6e2-e588fd8c1dae)

**Reply by OrdChaos** [reply]
我也很好奇
没套什么设定
用的阿里云的dns解析，阿里云自己的拨测没有问题，结果用别的（比如itdog）就告诉我境外一片解析失败
然后itdog换成阿里云dns服务器再测试，又正常了
发工单问了阿里云，说这是正常现象？

有什么支持境内境外分流的dns解析吗？（境内vps直连，境外netlify）
要不是阿里云dns支持我早换了

然后我也没想到扫不到是这个原因（
目前的话境内是一定可以访问的，境外得随缘
我抓紧换一下dns解析

**Reply by OrdChaos** [reply]
真的很无助

阿里云公共dns：
<img width="973" alt="image" src="https://github.com/user-attachments/assets/6bd4dc85-8017-4a93-8d82-8aeab095e9a1">

默认dns：
<img width="1015" alt="image" src="https://github.com/user-attachments/assets/28743b23-b584-4d0c-ad9a-99d2690e5108">

**Reply by OrdChaos** [reply, marked as answer]
已修复DNS解析
设置了DNSSEC就正常了
